packages/tools/pie-wrapper-react/__tests__/mocks/mock-component.ts

```diff
@@ -0,0 +1,15 @@
+// @ts-nocheck 
+    import * as React from 'react';
+    import { MockComponent as MockComponentReact} from './index';
```

Comment:
I can't find the `./index` file or the `MockComponent`

Reply:
the test actually just creates the template that a component would get - and references it's index. Since the test isn't a component, it doesn't actually directly link to anywhere 😄 I've actually removed this file though - it's auto generated through the test but isn't needed and probably ends up being more confusing than helpful